addons/remove-search-bar-autocomplete/addon.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Remove autocomplete from search bar",
+  "description": "Stops web browsers from autocompleting suggestions in the navigation bar's search bar.",
+  "tags": ["community"],
+  "credits": [
+    {
+      "name": "jbthepig",
+      "link": "https://scratch.mit.edu/users/jbthepig"
+    }
+  ],
```

Append a `/` to the end of users URIs
```suggestion
  "credits": [
    {
      "name": "jbthepig",
      "link": "https://scratch.mit.edu/users/jbthepig/"
    }
  ],
```